images/ipi-deprovision/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.svc.ci.openshift.org/openshift/hive-v4.0:hive
+LABEL maintainer="hongkliu@redhat.com"
+
+COPY ipi-deprovision/ipi-deprovision.sh  /usr/bin/ipi-deprovision.sh
```

Comment:
This assumes the context directory is `cmd`, I think this should be `cp cmd/ipi-deprovision/ipi-deprovision.sh destination`

Reply:
Done.